README.md

```diff
@@ -10,4 +10,61 @@ I will not be guarenting anything. I'll try to target both bluesky & mastodon, b
 
 ### This is not (and probably won't be for a while) a 100% accurate recreation of the twitter API
 
-most of this readme todo
\ No newline at end of file
+most of this readme todo
+
+## Compatibility
```

Comment:
More things, I like visual indicators, could we have it so:

⭐ = Active develop on
✅ = Works
❓ = Untested
⚠️ = Partly works (only use this for features which aren't implemented on a specific platform)
❌ = Doesn't work
💾 = Lost
🔒 = Won't fix (this would probably be used for twitter api 2.0, since that's a lil out of scope)

Reply:
+1.